doc/release-notes-19133.md

```diff
@@ -0,0 +1,7 @@
+## CLI
+
+A new `bitcoin-cli -generate` command, equivalent to RPC `generatenewaddress`
+followed by `generatetoaddress`, can generate blocks for command line testing
+purposes. This is a client-side version of the
+[former](https://github.com/bitcoin/bitcoin/issues/14299) `generate` RPC. See
+the help for details. (#19133)
```

Comment:
This doesn't produce a link (see markdown preview). I think you need to use `bitcoin/issues#14299`.

Reply:
Thanks @Sjors. Both of these links LGTM, so I'm not sure what you're seeing:

- https://github.com/bitcoin/bitcoin/pull/19357/files?short_path=551d874#diff-551d87469380922268db7eb7e5a618c1
- https://github.com/jonatack/bitcoin/blob/cli-generate-release-note/doc/release-notes-19133.md

Reply:
FWIW neither `#<issue>` or `bitcoin/issues#19133` produces a link in release notes (at least those published on the websites). They're markdown without github extensiosn. I don't think this is a problem though, people can look up the issue number of they want, it doesn't need to be full of links to github.

Reply:
I think the goal is so that the verbose paragraphs refer to the list of merged pull requests below (and again those refer to the actual pull requests on github?)